docs/introspection.rst

```diff
@@ -35,6 +35,10 @@ Advanced Usage
 The ``--indent`` option can be used to specify the number of indentation spaces to
 be used in the output. Defaults to `None` which displays all data on a single line.
 
+The ``--canonical`` option can be used to force the output to be sorted. This is
+useful if you want a reproducible canonical representation of your schema.
+Defaults to `False`, which displays keys in an arbitrary order.
```

Comment:
Is there any need to make this an option? It feels like this should be the standard behaviour, I'm not sure I'd ever want the keys in an undefined order.

Reply:
Good question. My initial instinct was to maintain backward compatibility - if you change behaviour it should default to the original behaviour.

But like you say, it's hard to imagine a use case where someone would want arbitrary order and specifically doesn't want sorted order.

If we're all happy with that, I'll drop the command option and the documentation, and just add the field to the JSON export.